core/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1504,8 +1503,8 @@ private void postIndexAsyncActions(String[] indices, List<CountDownLatch> inFlig
                     client().admin().indices().prepareFlush(indices).setIndicesOptions(IndicesOptions.lenientExpandOpen()).execute(
                             new LatchedActionListener<FlushResponse>(newLatch(inFlightAsyncOperations)));
                 } else if (isInternalCluster()) {
-                    internalCluster().getInstance(SyncedFlushService.class).attemptSyncedFlush(indices, IndicesOptions.lenientExpandOpen(),
-                            new LatchedActionListener<IndicesSyncedFlushResult>(newLatch(inFlightAsyncOperations)));
+                    client().admin().indices().syncedFlush(syncedFlushRequest(indices).indicesOptions(IndicesOptions.lenientExpandOpen()),
```

Comment:
We can get a transport client so my last version was actually wrong. I changes this now here to only do synced flush for versions >= 2.2: https://github.com/elastic/elasticsearch/commit/156ec36c1cd7685abe94e218589b261ba50ca9d5